internal/provider/resource_vault_cluster.go

```diff
@@ -82,9 +82,17 @@ func resourceVaultCluster() *schema.Resource {
 				Type:             schema.TypeString,
 				Optional:         true,
 				ValidateDiagFunc: validateSemVer,
-				ForceNew:         true,
+				// Given that Plus tier clusters will have Primary and Secondary on same version, changing the min vault version on primary shouldn't be possible. We need to handle this during apply
```

Comment:
I think this comment is a little unclear. What does the user need to take away here?